pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -436,11 +435,19 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 			// https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md
 			// Rates are in bits per second, burst values are in bits.
 			bandwidthParam.IngressRate = int(ingress.Value())
-			bandwidthParam.IngressBurst = math.MaxInt32 // no limit
+			if ingressBurst != nil {
+				bandwidthParam.IngressBurst = int(ingressBurst.Value())
+			} else {
+				bandwidthParam.IngressBurst = int(ingress.Value())
+			}
 		}
 		if egress != nil {
 			bandwidthParam.EgressRate = int(egress.Value())
-			bandwidthParam.EgressBurst = math.MaxInt32 // no limit
+			if egressBurst != nil {
+				bandwidthParam.EgressBurst = int(egressBurst.Value())
+			} else {
+				bandwidthParam.EgressBurst = int(egress.Value())
```

Comment:
This will break the default behaviour, since the bandwidth limitation is in a more or less experimental status this might be okay. We should at least ensure that the Release note is updated with a "Heads up" or an Action required. Also what is the pain point with letting `math.MaxInt32` the default when you can specify the burst rate now?

Reply:
I think we needn't specify burst rate in most situation, so let default burst equal rate is better.